.gitignore

```diff
@@ -122,4 +123,7 @@ captures/
 
 # Intellij
 *.iml
-.idea/workspace.xml
\ No newline at end of file
+.idea/workspace.xml
+
+# Flutter module
+!ostorlab_insecure_flutter_app/**
```

Comment:
Why adding this?

Reply:
There is conflict between `.gitignore` from the android project and the one from the flutter module, I had to force include `ostorlab_insecure_flutter_app` and let it use its own `.gitignore`